workflow/controller/informer/tolerant_cluster_workflow_template_informer.go

```diff
@@ -3,6 +3,8 @@ package informer
 import (
 	"time"
 
+	"github.com/argoproj/argo-workflows/v3/workflow/common"
```

Comment:
this should be in the same block as the other internal imports below (starting at line 15)